File: StoryDomain/getUserStoriesService - copia/Models/Story.js
var mongoose = require('mongoose');
var Schema = mongoose.Schema;

var StorySchema = Schema({
    image: { type: String, required: true },
    user: { type: Schema.ObjectId, ref: 'user', require: true },
    exp: { type: Date, require: true },
    createdAt: { type: Date, default: Date.now },
});

module.exports = mongoose.model('story', StorySchema);

File: StoryDomain/CreateStoryService/Model/Story.go
package Model

import (
	"time"
	"go.mongodb.org/mongo-driver/bson/primitive"
)


type Story struct {
	// ID        primitive.ObjectID `bson:"_id,omitempty" json:"id"`
	Image     string             `bson:"image" json:"image"`
	User      primitive.ObjectID `bson:"user,omitempty" json:"user"`
	Exp       time.Time          `bson:"exp" json:"exp"`
	CreatedAt time.Time          `bson:"createdAt,omitempty" json:"createdAt,omitempty"`
}
